content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._en.md

```diff
@@ -1,36 +1,136 @@
 ---
 layout: help
-title: How to add images of your state-issued ID
+title: How to add images of your driver’s license or state ID card
 category: verify-your-identity
 permalink: /help/verify-your-identity/how-to-add-images-of-your-state-issued-id/
 order: 3
 redirect_from:
   - /en/help/verify-your-identity/how-to-add-images-of-your-state-issued-id/
+do_list: 
+  - Do use a high resolution camera like a smartphone or tablet camera. Your computer webcam may not take clear photos.
+  - Do use a solid, dark background
+  - Do take your photos in a well-lit area with indirect light
+  - Do check that ID barcodes are not damaged or dirty
+  - Do try to hold your phone still while taking the photos. It may help to prop your arms on the table to steady yourself.
 ---
 
-A clear picture of your state-issued ID is required to complete the identity verification process. For best results, we recommend using a phone to automatically take a photo of the front and back of your ID. If you are verifying your identity on a computer, you will be able to switch to a phone for this part of the process. You are not required to upload a picture of yourself or take a selfie for identity verification.
+{% capture alert_link %}
+  <a href="/help/verify-your-identity/accepted-state-issued-identification/">valid driver's license or state ID card</a>
+{% endcapture %}
 
-## Tips for taking photos of your ID with a camera
+{% capture alert_content %}
+  <strong>
+    If you do not have a {{ alert_link }} you cannot use Login.gov for identity verification.
+  </strong>
+  Please contact the partner agency’s help center to find out what you can do instead.
+{% endcapture %}
 
-* Do use a high resolution camera (8 megapixel or higher). Most phone or tablet cameras will work, your computer webcam may not. We recommend you use your mobile device.
-* Do use a solid, dark background.
-* Do show the edges of the ID in the photo.
-* Do take a clear photo without shadows, glares, reflections or blur.
-* Do check that barcodes are not damaged or dirty.
-* Do take the photo in a well lit area with indirect light.
-* Don’t use your flash.
-* Don’t crop the background.
+{%
+  include alert.html
+  content=alert_content
+  type='error'
+  role='alert'
+%}
 
-## Tips for uploading or scanning your ID:
+<div class="margin-top-5">
+  <span class="usa-tag usa-tag--informative">Recommended</span>
+</div>
 
-* Follow the same tips for taking photos of your ID with a camera
+<div class="margin-top-2">
+  <h2 id="use-a-phone-to-take-photos-of-the-front-and-back-of-your-id" class="margin-0">
```

Comment:
I saw the commit where this ID was added and not really sure about it. I noticed the English version of the page has an anchor link for "tips for taking photos with a phone", but the equivalent doesn't exist in the French or Spanish versions of the page. I would expect an equivalent link exist in those translated pages, and that it would point to the (default) translated anchor ID for the equivalent section.

Reply:
Can you say more about this?

I'm not sure what the default ID generation is. I also don't fully understand the comment about the fragment.

Reply:
When using Markdown headings, it'll automatically generate an ID based on the text of the heading, e.g.

```
## Hello world
```

➡️ 

```
<h2 id="hello-world">Hello world</h2>
```

I see that that doesn't happen here because we're manually crafting the HTML `<h2>` tag.

So what I had in mind was something like this:

```diff
diff --git a/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._en.md b/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._en.md
index cb169a3c..61346fe7 100644
--- a/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._en.md
+++ b/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._en.md
@@ -38,7 +38,3 @@ do_list:
 
-<div class="margin-top-2">
-  <h2 id="phone-tips" class="margin-0">
-    Use a phone to take photos of the front and back of your ID
-  </h2>
-</div>
+## Use a phone to take photos of the front and back of your ID
 
@@ -113,3 +109,3 @@ If you are using a computer to verify your identity, you will be able to switch
 ## If you don’t have access to a phone with a camera, you can upload a file or use a scanner
-* Follow the same [tips for taking photos with a phone](#phone-tips)
+* Follow the same [tips for taking photos with a phone](#use-a-phone-to-take-photos-of-the-front-and-back-of-your-id)
 * Save each file as a JPG or PNG. You cannot use a PDF file. Look at the name of your file and it should have an abbreviation after the title (ex: JohnDoeID_Front.jpg)
diff --git a/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._es.md b/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._es.md
index be497c68..f238d28e 100644
--- a/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._es.md
+++ b/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._es.md
@@ -36,7 +36,3 @@ do_list:
 
-<div class="margin-top-2">
-  <h2 id="phone-tips" class="margin-0">
-      Con un teléfono, realice fotografías del anverso y reverso de su documento de identidad
-  </h2>
-</div>
+## Con un teléfono, realice fotografías del anverso y reverso de su documento de identidad
 
@@ -111,3 +107,3 @@ Si está utilizando una computadora para verificar su identidad, puede cambiar a
 ## Si no tienes acceso a un teléfono con cámara, puedes subir un archivo o utilizar un escáner.
-* Sigue los mismos [consejos para tomar fotografías con un teléfono](#phone-tips)
+* Sigue los mismos [consejos para tomar fotografías con un teléfono](#con-un-teléfono-realice-fotografías-del-anverso-y-reverso-de-su-documento-de-identidad)
 * Guarda cada archivo como JPG o PNG. No puedes utilizar un archivo PDF. Fíjate en el nombre de tu archivo y debe tener una abreviatura después del título (por ejemplo: JohnDoeID_Front.jpg)
diff --git a/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._fr.md b/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._fr.md
index 5b338584..dbe99d9a 100644
--- a/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._fr.md
+++ b/content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._fr.md
@@ -36,7 +36,3 @@ do_list:
 
-<div class="margin-top-2">
-  <h2 id="phone-tips" class="margin-0">
-      Utilisez un téléphone pour prendre des photos du recto et du verso de votre pièce d’identité
-  </h2>
-</div>
+## Utilisez un téléphone pour prendre des photos du recto et du verso de votre pièce d’identité
 
@@ -111,3 +107,3 @@ Si vous utilisez un ordinateur pour vérifier votre identité, vous pourrez pass
 ## Si vous n'avez pas accès à un téléphone équipé d'une caméra, vous pouvez téléverser un fichier ou utiliser un scanner
-* Suivez les mêmes [conseils pour prendre des photos avec un téléphone](#phone-tips).
+* Suivez les mêmes [conseils pour prendre des photos avec un téléphone](#utilisez-un-téléphone-pour-prendre-des-photos-du-recto-et-du-verso-de-votre-pièce-didentité).
 * Utilisez un fond solide et sombre.
```

I suspect you had to do the manual HTML to get the desired spacing between the tag and heading, so I'm not sure if that's technically possible, since the default heading styles are pretty strong (`margin-top` utility classes don't work). A negative bottom margin on the tag (`margin-bottom-neg-2`) could be an option, but I think the approach you have here currently is fine too.

Reply:
The point about the translated fragment is:

An English-speaking user who sees the following URL could understand that it'll lead them to a subsection specifically about phone tips:

https://login.gov/help/verify-your-identity/how-to-add-images-of-your-state-issued-id/#phone-tips

A Spanish-speaking user who sees the following URL wouldn't be able to make as much sense of "phone tips" if they don't understand English:

https://login.gov/es/help/verify-your-identity/how-to-add-images-of-your-state-issued-id/#phone-tips

They would be able to make more sense of it as the following, per the diff above:

https://login.gov/es/help/verify-your-identity/how-to-add-images-of-your-state-issued-id/#con-un-teléfono-realice-fotografías-del-anverso-y-reverso-de-su-documento-de-identidad

Though now as I write this I realize this is already a problem with how the paths are in English 😓 ([related Slack discussion](https://gsa-tts.slack.com/archives/C01710KMYUB/p1693242056792359?thread_ts=1693233563.852139&cid=C01710KMYUB))

Reply:
Thanks for spelling all that out.
- I didn't realize that Jekyll auto-generates IDs for headers.
- You're correct: I had to do it this way for spacing reasons.
- I did actually implement the `#con-un-teléfono-realice...` ID the way you're suggesting, but the weirdness with the rest of the URL being in English stopped me from pursuing it. I decided it was more consistent when the whole URL was in english, rather than suddenly having a spanish part on the end. (👁️ on that thread)